src/App.css

```diff
@@ -109,5 +107,5 @@
 
 .Date {
   position: relative;
-  left: 30%;
+  right: 10%;
 }
```

Comment:
This works really nicely and responsively on different mobile sizes which is really impressive. It skews to the left on browser, but this is something you can come back to later.